.circleci/config.yml

```diff
@@ -242,6 +254,15 @@ jobs:
             - "~/.rustup"
             - "~/.cargo"
 
+  gofmt:
```

Comment:
`gofmt` should currently be checked as part of the linter?
Isn't it?

Reply:
Yes, but no one bothers to look at it

(I mean, it fails here too..)